File: Desktop/andela/bootcamp/Homeday1/fizzbuzz.js
function fizzBuzz(num){ // this code works fine and passes all tests on andelab
	// divisible by only 3
	if ((num%3 === 0) && (num%5 !== 0)){
		return 'Fizz';
	}
	// divisible by only 5
	else if((num%5 === 0) && (num%3 !== 0)){
		return 'Buzz';
	}
	// divisible by both 3 and 5
	else if((num%5 === 0) && (num%3 === 0)){
		return 'FizzBuzz';
	}
	// divisible by neither 3 nor 5
	else if((num%5 !== 0) && (num%3 !== 0)){
		return num;
	}
}

File: FizzBuzz.py
def fizz_buzz(num):
    # divisible by only 3
    if (num%3 == 0) and (num%5 != 0):
        return 'Fizz'

    # divisible by only 5
    elif (num%3 != 0) and (num%5 == 0):
        return 'Buzz'

    # divisible by both 3 and 5
    elif (num%3 == 0) and (num%5 == 0):
        return 'FizzBuzz'

    # divisible by neither 3 nor 5
    elif (num%3 != 0) and (num%5 != 0):
        return num

n = 101
print fizz_buzz(n)